_rules/image-accessible-name-23a2a8.md

```diff
@@ -35,7 +35,7 @@ htmlHintIgnore:
 
 ## Applicability
 
-The rule applies to HTML `img` elements or any HTML element with the [semantic role][] of `img` that is [included in the accessibility tree][].
+The rule applies to HTML `img` elements or any HTML element with the [semantic role][] of `img` that is [included in the accessibility tree][] and displays an image.
```

Comment:
I think "displays an image" need to be clarified.
For example `<img src="this.url.does.not.exists" />` arguably does not display an image, but I think this is not what you intend to rule out.

Reply:
How about changing:

> and displays an image

to
> and renders an image or an omitted image indicator

based on the rendering description here https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element:attr-img-src-5

Is that description precise enough?

A couple of options that won't work:

1) Using the [represents nothing](https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element:attr-img-src-5) definition from HTML. I don't think that works if there's a CSS background-image and no src.

2) Using the [valid non-empty URL potentially surrounded by spaces](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#valid-non-empty-url-potentially-surrounded-by-spaces) definition from HTML. That doesn't work for inline SVG.

Reply:
How about this?
> it's [current request][]'s [state][image request state] is not [completely available][]; or
I used it here: https://github.com/act-rules/act-rules.github.io/pull/1213/files#diff-f7f0c7b0159f5cafa41a1983a6d9389aR30